modules/ROOT/pages/_partials/commons/common-prebuilt-database.adoc

```diff
@@ -10,24 +10,21 @@
 == Overview
 
 
-{cbl-te}_'s_ support for pre-built databases means you can pre-load your app with data instead of syncing it down from {sgw-te} during startup.
-This might benefit, for example, a mobile application developer striving to minimize consumer wait time (arising from data setup) on initial install and launch of the application.
+{cbl-te} supports  pre-built databases. You can pre-load your app with data instead of syncing it from {sgw-te} during startup to minimize consumer wait time (arising from data setup) on initial install and launch of the application. 
 
-Avoiding an initial bulk sync will help reduce startup time and network transfer costs, leading to a better consumer experience.
-It is typically more efficient to download bulk data using the http/ftp stream  employed during the application installation than to install a smaller application bundle and then have to use a replicator to pull-in the bulk data.
+Avoiding an initial bulk sync reduces startup time and network transfer costs. It is typically more efficient to download bulk data using the http/ftp stream employed during the application installation than to install a smaller application bundle and then use a replicator to pull-in the bulk data.
 
-This prepackaging of the data typically applies to public/shared, non-user specific data that is mostly static.
-Even if the data is not static, you can still benefit from preloading it and only syncing the changed documents on startup
+Pre-loaded data is typically public/shared, non-user-specific data that is static. Even if the data is not static, you can still benefit from preloading it and only syncing the changed documents on startup.
```

Comment:
new line etc.

"mostly static" seems to be different from "static".
I realise the caveat on next line kinda explains that, but calling it out isn't terrible.

Reply:
mostly is a wishy washy word - we talk about non-static data in the next sentence.